orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationAiModel.java

```diff
@@ -0,0 +1,35 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.LLMModuleConfig;
+import java.util.Map;
+import javax.annotation.Nonnull;
+
+/** Large language models available in Orchestration. */
```

Comment:
We should probably add this is what is available on EU12? EU10? Because orchestration only has access to the models that are available in that BTP region

Reply:
If I add all models I have to remove my test, is this ok?

Reply:
> is this ok?

As long as Orchestration Service does not provide an endpoint to list all available models. I'm afraid this can't be tested properly.